### DISCUSSION TITLE
Using alternative option for LLP sender ConnectorType from MirthConnect version 2.2.x.

### DISCUSSION THREAD

**Discussion by gpatil-github** [nextgenhealthcare/connect#6217] [Q&A] [answered]
We have a mirth connect version 4.4.2 and for legacy usage we have 2.2.X in 2.2.X we have used connectorType as LLP Sender, We wanted move that to 4.4.2 but in 4.4.2 we don't have option for configure LLP sender

Can we use other ConnectorType instead like TCPSender or hTTPSender withpur changes needed at client sise?

Client has VPN with our cloud network and from our cloud Network MirthConnect we push message via VPN to HL7 client engine.

We are also thinking about replacing the MirthConnect with our inhouse component which can be utilized for this workflow. The idea is to have a REST based component in the VPN setup with client and push the message via the VPN setup to customer's provided IP address/URL endpoint.

**Comment by pacmano1** [comment, marked as answer]
Just use a TCP sender of type MLLP.  you should probably take a closer look at the user manual and the upgrade guides to understand what has changed between releases.  

![image](https://github.com/nextgenhealthcare/connect/assets/44065187/59ddd5b4-84ee-4bfe-b416-c4d14c6174af)
